docs/tutorial/force_calibration.rst

```diff
@@ -1,380 +1,375 @@
+Fast detectors have the ability to respond much faster to incoming light resulting in no visible filtering effect in the frequency range we are fitting.
+This means that for a fast detector, we do not need to include a filtering effect in our model.
+We can omit this effect by passing `fast_sensor=True` to the calibration models or to :func:`~lumicks.pylake.calibrate_force()`.
+Note however, that this makes using the hydrodynamically correct model critical, as the Lorentzian doesn't actually capture the data very well.
+Let's show this by loading a dataset acquired on a fast sensor::
 
-.. math::
+    f = lk.File("fast_measurement_25%.h5")
 
-    P_\mathrm{hydro}^\mathrm{active}(f) = \delta \left(f - f_\mathrm{drive}\right) \frac{\left(A f_\mathrm{drive} \left|\gamma / \gamma_0\right|\right)^2}{2 \left(\left(f_{c,0} + f \mathrm{Im}(\gamma/\gamma_0) - f^2/f_{m, 0}\right)^2 + \left(f \mathrm{Re}(\gamma / \gamma_0)\right)^2\right)}
+    shared_parameters = {
+        "force_voltage_data": decalibrate(f.force2y).data,
+        "bead_diameter": 4.38,
+        "temperature": 25,
+        "sample_rate": volts.sample_rate,
+        "fit_range": (1e2, 23e3),
+        "excluded_ranges": ([190, 210], [13600, 14600])
+    }
 
-We can also include a distance to the surface like before.
-This results in the expression for `\gamma` becoming dependent on the distance to the surface.
-This uses the same expression as listed in the section on the hydrodynamically correct model.
+    plt.figure(figsize=(13, 4))
+    plt.subplot(1, 3, 1)
+    fit = lk.calibrate_force(**shared_parameters, hydrodynamically_correct=False, fast_sensor=False, num_points_per_block=200)
+    fit.plot()
+    plt.title(f"Lorentzian + Slow (kappa={fit['kappa'].value:.2f})")
+    plt.subplot(1, 3, 2)
+    fit = lk.calibrate_force(**shared_parameters, hydrodynamically_correct=False, fast_sensor=True, num_points_per_block=200)
+    fit.plot()
+    plt.title(f"Lorentzian + Fast (kappa={fit['kappa'].value:.2f})")
+    plt.subplot(1, 3, 3)
+    fit = lk.calibrate_force(**shared_parameters, hydrodynamically_correct=True, fast_sensor=True, num_points_per_block=200)
+    fit.plot()
+    plt.title(f"Hydro + Fast (kappa={fit['kappa'].value:.2f})")
+    plt.tight_layout()
+
+.. image:: figures/force_calibration/fast_sensors.png
+
+Note how the fit with the Lorentzian model with the filtering effect enabled seems to fit the data quite well.
+However, the apparent quality of a fit can be deceiving.
+When the `fast_sensor` flag is enabled, it can be seen that the Lorentzian model doesn't actually describe the data.
+Switching to the hydrodynamically correct model results in a superior fit to the power spectrum.
+
+In the case of the Lorentzian with a detector filtering effect, the fitting procedure has used the parameters of the filter to fit some of the high frequency attenuation.
+With the diode model disabled, we obtain a very biased fit.
```

Comment:
here are you using "Lorentzian" to refer to passive calibration? If so, I think we should be consistent and just use passive calibration throughout

Reply:
No, I use Lorentzian to mean the simple model. I think we should actual hold off on calling it a Lorentzian in the tutorial (and save that for the theory section). I've started referring to it as simple model.

I have also updated the description a bit, and added a question. While I normally try to avoid adding questions like this in explanations, I think in this case it is warranted:

```
Note how the power spectral fit with the simple model seems to fit the data quite well as long as we also include the filtering effect.
However, the apparent quality of a fit can be deceiving.
Considering that this dataset was acquired on a fast sensor, we should omit the filtering effect.
When the `fast_sensor` flag is enabled, it can be seen that the simple model doesn't actually describe the data.
Switching to the hydrodynamically correct model results in a superior fit to the power spectrum.

So what is happening here? Why did the first fit look good?
When we fit the power spectrum with the simple model and include the filtering effect, the fitting procedure uses the parameters that characterize the filter to fit some of the high frequency attenuation.
With the filtering effect disabled, we obtain a very biased fit because the model fails to fit the data.
```